sdk/lib/crt/crt.cmake

```diff
@@ -581,9 +581,9 @@ endif()
 set_source_files_properties(${CRT_ASM_SOURCE} PROPERTIES COMPILE_DEFINITIONS "__MINGW_IMPORT=extern;USE_MSVCRT_PREFIX;_MSVCRT_LIB_;_MSVCRT_;_MT;CRTDLL")
 add_asm_files(crt_asm ${CRT_ASM_SOURCE})
 
-if(CMAKE_C_COMPILER_ID STREQUAL "Clang")
-    #FIXME: http://llvm.org/bugs/show_bug.cgi?id=19027
-    set_property(SOURCE except/cpp.c APPEND_STRING PROPERTY COMPILE_FLAGS " -no-integrated-as")
+if(USE_CLANG_CL)
+    # clang-cl is missing pragma function support
+    set_property(SOURCE stdlib/rot.c APPEND_STRING PROPERTY COMPILE_FLAGS " /fallback")
 endif()
```

Comment:
Does it matter ? Did you try to comment the pragma out ?

Reply:
Without pragma function these intrinsics won't be generated as functions and you'll get unresolved references to them in the crt, and the same thing happens if you comment them out for clang-cl because it errors about them being redefined intrinsics/builtins. We need to find a way to get clang-cl to emit these.

Reply:
You probably would know this already, but specifying `/fallback` here will break compilation with clang on non-Windows platforms, where the fallback CL.exe is not available. I don't know how high-priority compiling ReactOS with Clang on a non-Windows platform is for you, but I just wanted to let you know about this.

Unfortunately, it [doesn’t look like](https://github.com/llvm-mirror/clang/blob/a54d18761bea291cfe7735f73d4b45ff23b7a835/lib/Sema/SemaDecl.cpp#L12226-L12232) these errors can be disabled, short of patching clang.